doc/appendix-de-kassensichv/reference-tables/reference-tables.md

```diff
@@ -279,3 +279,474 @@ For definitions regarding national laws, please refer to the appropriate appendi
 
 *Table 15. Type of Signature: ftSignatureType*
 
+### DSFinV-K export
```

Comment:
Should these changes be part of this PR, or of #56?

Reply:
Hi Tom, true, thank you for the hint. I removed the dsfivk related changes. Can you pls. check it again?

Reply:
Looks good to me, thank you!